the_big_bang.ipynb

```diff
@@ -0,0 +1,295 @@
+{
```

Comment:
<p><code>nltk</code>&nbsp;is not working with the basic tools on freeze, that's why I used&nbsp;<code>die</code>.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SygniaLabs/backend-dev-task/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>